.nycrc

```diff
@@ -0,0 +1,3 @@
+{
+  "exclude": ["**/*/*.spec.ts"]
+}
```

Comment:
Trivial: newline

Reply:
in: https://github.com/sonatype-nexus-community/auditjs/pull/141/commits/47b566b5298f5f37732e706a52596faebad0333f